trajopt_sco/package.xml

```diff
@@ -9,6 +9,7 @@
   <depend>osqp</depend>
   <depend>trajopt_utils</depend>
   <depend>libjsoncpp-dev</depend>
+  <depend>osqp</depend>
```

Comment:
osqp is already added at line 9. This may be the issues with ci.

Reply:
Whoops, good catch!